tests/distributions/conftest.py

```diff
@@ -262,20 +272,26 @@ def __init__(self, loc, scale):
             examples=[
                 {'dim': 3, 'concentration': [1.], 'test_data':
                     [[[1.0000, -0.8221,  0.7655], [-0.8221,  1.0000, -0.5293], [0.7655,  -0.5293,  1.0000]],
-                     [[1.0000, -0.5345, -0.5459], [-0.5345,  0.9999, -0.0333], [-0.5459, -0.0333,  1.0001]],
+                     [[1.0000, -0.5345, -0.5459], [-0.5345,  1.0000, -0.0333], [-0.5459, -0.0333,  1.0000]],
                      [[1.0000, -0.3758, -0.2409], [-0.3758,  1.0000,  0.4653], [-0.2409,  0.4653,  1.0000]],
                      [[1.0000, -0.8800, -0.9493], [-0.8800,  1.0000,  0.9088], [-0.9493,  0.9088,  1.0000]],
-                     [[1.0000,  0.2284, -0.1283], [0.2284,   1.0001,  0.0146], [-0.1283,  0.0146,  1.0000]]]},
+                     [[1.0000,  0.2284, -0.1283], [0.2284,   1.0000,  0.0146], [-0.1283,  0.0146,  1.0000]]]},
                 ]),
     Fixture(pyro_dist=dist.LKJCorrCholesky,
             examples=[
-                {'d': 3, 'eta': [1.], 'test_data':
-                    [[[1.0000,  0.0000,  0.0000], [-0.8221,  0.5693,  0.0000], [0.7655,   0.1756,  0.6190]],
-                     [[1.0000,  0.0000,  0.0000], [-0.5345,  0.8451,  0.0000], [-0.5459, -0.3847,  0.7444]],
-                     [[1.0000,  0.0000,  0.0000], [-0.3758,  0.9267,  0.0000], [-0.2409,  0.4044,  0.8823]],
-                     [[1.0000,  0.0000,  0.0000], [-0.8800,  0.4750,  0.0000], [-0.9493,  0.1546,  0.2737]],
-                     [[1.0000,  0.0000,  0.0000], [0.2284,   0.9736,  0.0000], [-0.1283,  0.0451,  0.9907]]]},
-                ]),
+                {
+                    'd': 3,
+                    'eta': [1.],
+                    'test_data': [
+                        [[1.0, 0.0, 0.0],
+                         [-0.17332135, 0.98486533, 0.0],
+                         [0.43106407, -0.54767312, 0.71710384]],
+                        [[1.0, 0.0, 0.0],
+                         [-0.31391555, 0.94945091, 0.0],
+                         [-0.31391296, -0.29767500, 0.90158097]],
+                    ],
+                },
+            ]),
```

Comment:
the old low-precision test data was failing the new `test_support_shape`, so I created new data